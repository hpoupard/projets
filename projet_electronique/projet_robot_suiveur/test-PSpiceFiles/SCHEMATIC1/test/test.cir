** Profile: "SCHEMATIC1-test"  [ C:\Users\hpoupard\Desktop\projet_robot_suiveur\test-PSpiceFiles\SCHEMATIC1\test.sim ] 

** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
* From [PSPICE NETLIST] section of C:\Cadence\SPB_16.5\tools\PSpice\PSpice.ini file:
.lib "nom.lib" 

*Analysis directives: 
.DC DEC I_I1 0.1u 10u 5 
.PROBE V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 


.END
