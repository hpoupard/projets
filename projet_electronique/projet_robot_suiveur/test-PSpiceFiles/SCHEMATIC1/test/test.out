
**** 11/06/17 11:14:34 ****** PSpice 16.5.0 (April 2011) ****** ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ C:\Users\hpoupard\Desktop\projet_robot_suiveur\test-PSpiceFiles\SCHEMATIC1\test.sim ] 


 ****     CIRCUIT DESCRIPTION


******************************************************************************




** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
* From [PSPICE NETLIST] section of C:\Cadence\SPB_16.5\tools\PSpice\PSpice.ini file:
.lib "nom.lib" 

*Analysis directives: 
.DC DEC I_I1 0.1u 10u 5 
.PROBE V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 



**** INCLUDING SCHEMATIC1.net ****
* source TEST
X_U1-1         0 N00201 VCC 0 N00238 lmv321/NS
R_R1         N00201 N00238  3.3Meg TC=0,0 
I_I1         N00201 0 DC 0Adc  
V_V1         VCC 0 5

**** RESUMING test.cir ****
.END

**** 11/06/17 11:14:34 ****** PSpice 16.5.0 (April 2011) ****** ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ C:\Users\hpoupard\Desktop\projet_robot_suiveur\test-PSpiceFiles\SCHEMATIC1\test.sim ] 


 ****     Diode MODEL PARAMETERS


******************************************************************************




               X_U1-1.DMOD1    X_U1-1.DMOD2    
          IS   10.000000E-15   10.000000E-18 


**** 11/06/17 11:14:34 ****** PSpice 16.5.0 (April 2011) ****** ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ C:\Users\hpoupard\Desktop\projet_robot_suiveur\test-PSpiceFiles\SCHEMATIC1\test.sim ] 


 ****     BJT MODEL PARAMETERS


******************************************************************************




               X_U1-1.NPN1     X_U1-1.PNP1     
               NPN             PNP             
       LEVEL    1               1            
          IS    1.000000E-15    1.000000E-15 
          BF  100             100            
          NF    1               1            
          BR    1               1            
          NR    1               1            
         ISS    0               0            
          RE    0               0            
          RC    0               0            
         CJE    0               0            
         VJE     .75             .75         
         CJC    0               0            
         VJC     .75             .75         
         MJC     .33             .33         
        XCJC    1               1            
         CJS    0               0            
         VJS     .75             .75         
          KF    0               0            
          AF    1               1            
          CN    2.42            2.2          
           D     .87             .52         


**** 11/06/17 11:14:34 ****** PSpice 16.5.0 (April 2011) ****** ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ C:\Users\hpoupard\Desktop\projet_robot_suiveur\test-PSpiceFiles\SCHEMATIC1\test.sim ] 


 ****     Resistor MODEL PARAMETERS


******************************************************************************




               X_U1-1.MRAX     X_U1-1.MRIB     X_U1-1.MRISC    
           R    1               1               1            
         TC1   10.000000E-06    2.837130E-03   -3.000000E-03 



          JOB CONCLUDED

**** 11/06/17 11:14:34 ****** PSpice 16.5.0 (April 2011) ****** ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ C:\Users\hpoupard\Desktop\projet_robot_suiveur\test-PSpiceFiles\SCHEMATIC1\test.sim ] 


 ****     JOB STATISTICS SUMMARY


******************************************************************************



  Total job time (using Solver 1)   =         .08
